actors/miner/tests/miner_actor_test_wpost.rs

```diff
@@ -90,7 +92,390 @@ fn basic_post_and_dispute() {
 }
 
 #[test]
-fn invalid_submissions() {}
+fn invalid_submissions() {
+    let period_offset = ChainEpoch::from(100);
+    let precommit_epoch = ChainEpoch::from(1);
+
+    let mut h = ActorHarness::new(period_offset);
+    let mut rt = h.new_runtime();
+    rt.epoch = precommit_epoch;
+    rt.balance.replace(TokenAmount::from(BIG_BALANCE));
+
+    h.construct_and_verify(&mut rt);
+
+    let sectors = h.commit_and_prove_sectors(&mut rt, 1, DEFAULT_SECTOR_EXPIRATION, vec![], true);
+    let sector = sectors[0].clone();
+    let pwr = miner::power_for_sector(h.sector_size, &sector);
+
+    // Skip to the due deadline.
+    let state = h.get_state(&rt);
+    let (dlidx, pidx) = state.find_sector(&rt.policy, &rt.store, sector.sector_number).unwrap();
+    let dlinfo = h.advance_to_deadline(&mut rt, dlidx);
+
+    // Invalid deadline.
+    {
+        let partition = miner::PoStPartition { index: pidx, skipped: make_empty_bitfield() };
+        let params = miner::SubmitWindowedPoStParams {
+            deadline: rt.policy.wpost_period_deadlines,
+            partitions: vec![partition],
+            proofs: make_post_proofs(h.window_post_proof_type),
+            chain_commit_epoch: dlinfo.challenge,
+            chain_commit_rand: Randomness(Vec::from(b"chaincommitment".clone())),
```

Comment:
```suggestion
            chain_commit_rand: Randomness(b"chaincommitment".to_vec()),
```

Reply:
ah, ok, that's nicer. I'll change it everywhere I guess.

Reply:
done.